SFTPAdapterInterface/conanfile.py

```diff
@@ -0,0 +1,17 @@
+from conans import ConanFile, tools
+
+class SFTPAdapterInterfaceConan(ConanFile):
+	name = "SFTPAdapterInterface"
+	description = "Interface of library-agnostic API for C++ to work with SFTP"
+	author = "CSW <csw@werfen.com>"
+	topics = ("conan", "sftp", "adapter", "wrapper")
+	license = "MIT"
+	generators = "visual_studio"
```

Comment:
I know it doesn't cause problems so we can leave it, but in theory the interface wouldn't need any generator, would it?

Reply:
Probably not. It might made sense if it had dependencies but is not the case.
Since packet is already generated and this pose no problem, I would leave it as is but take into consideration for future recipes.